src/binding/abi/mpi_abi.h

```diff
@@ -425,7 +425,7 @@ enum {
 };
 
 /* File Operation Constants */
-#define MPI_DISPLACEMENT_CURRENT       ((MPI_Offset)-54278278)
+#define MPI_DISPLACEMENT_CURRENT       ((MPI_Offset)-1)
```

Comment:
Tests look OK with this value. Should be propose a change to https://github.com/mpi-forum/mpi-standard/pull/977 for next week's second vote?

Reply:
I am for it

Reply:
Let me drop the last commit (https://github.com/pmodels/mpich/pull/7247/commits/05c3ec93bfc199195a6909ca1e069f5846880df1) and test. If test pass, it means we can maintain both MPICH ABI and whatever negative value MPI ABI will use.